eunkyung/app.js

```diff
@@ -0,0 +1,53 @@
+const http = require('http')
+const express = require('express')
+const cors = require('cors')
+const dotenv = require('dotenv')
+const morgan = require('morgan')
+
+dotenv.config()
+
+const { DataSource } = require('typeorm')
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => { console.log('data server has been initialized!') })
```

Comment:
catch 구문을 통해 데이터베이스에 connection이 실패했을 경우 에러를 반환할 수 있도록 수정해주세요!
```suggestion
myDataSource.initialize()
    .then(() => {
      console.log('data server has been initialized!')
    })
    .catch(err => {
      console.log("Failed to Connect Database")
    })
```